README.md

```diff
@@ -1 +1,2 @@
-# chrome-watcher
\ No newline at end of file
+# chrome-watch
+This extension displays how much time you spend on various websites in a day.
```

Comment:
I've partially updated this - at 00:00 I reset all data so it's in a day, not over a period of time.